### PR TITLE
White Bread Recipe

### PR DESCRIPTION
**Pull request by michaelc-xyz**

### CODE REVIEW

File: src/white-bread.md [hunk 1]
```diff
@@ -0,0 +1,38 @@
+# White Sandwich Bread
+
+![White Bread](pix/white-bread.webp)
+
+Simple white bread. Like the bread you get from the grocery store but 100x better.
+
+- ⏲️ Prep time:     10 min
+- ⬆️ Proofing time: 1.5 - 2.5 hrs
+- 🍞Baking time:   35 min - 1 hr
```
Comment (zyansheep) [inline src/white-bread.md]: add space before baking time and after emoji

File: src/white-bread.md [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+
+## Ingredients
+
+- 3/4 cup (174g) water
+- 1/2 cup (125g) whole milk
```
Comment (zyansheep) [inline src/white-bread.md:14]: be consistent using 3/4 or ¾

File: src/white-bread.md [hunk 3]
```diff
@@ -0,0 +1,38 @@
...
+- 1 packet (9g) active dry yeast (or 6.75g - you can round up to 7g if your scale doesn't have decimals -  instant yeast)
```
Comment (zyansheep) [inline src/white-bread.md:15]: extra space after second `-`. also, what is the difference between 9g and 6.75g yeast?

File: src/white-bread.md [hunk 4]
```diff
@@ -0,0 +1,38 @@
...
+- 3 cups (440g) bread flour (or 408g all purpose flour + 32g vital wheat gluten)
+- 1 ½ tsp (8g) fine sea salt
+- 1 tbsp (21g) granulated sugar
+- 3 tbsp (42g) unsalted butter, softened (you can substitute with 31.5g of flavorless oil like sunflower oil, vegetable oil, coconut oil, light olive oil etc.)
```
Comment (zyansheep) [inline src/white-bread.md:19]: perhaps specify to use vegetable oil? or not.. I wonder what bacon grease would taste like in white bread...

File: src/white-bread.md [hunk 5]
```diff
@@ -0,0 +1,38 @@
...
+9. Preheat oven to 350ºF. *Depending on how fast your oven heats up, you can do this during the last 10 - 15 mins of your second proof. This way your dough won't be in danger of over-proofing while you wait for your oven to finish pre-heating*
+10. Place loaf pan in the oven and bake  at 350ºF for 35 to 40 mins or until golden brown. Take loaf out of pan and place on a cooling rack. You can test if the loaf is properly cooked by tapping the bottom. If it sounds hollow, it's done. If there's no sound, the inside might be undercooked. Return the loaf in the oven for another 5 to 10 mins (monitor closely to prevent burning). Let your loaf cool completely before slicing.
```
Comment (zyansheep) [inline src/white-bread.md:32]: I like all the detail in this recipe! Bread is a hard thing to make right...

**Review by zyansheep — CHANGES_REQUESTED** [review]

File: src/white-bread.md (hunk 3 above)
Comment (endlesslake) [reply src/white-bread.md:15]: That is 9g of active dry yeast or 6.75g of instant yeast. Two different types of yeast

File: src/white-bread.md (hunk 5 above)
Comment (endlesslake) [reply src/white-bread.md:32]: I straight up copied this recipe from a comment on a YouTube video

**Pull request closed without merging**